docs/src/pages/components/Link/WebLinksPage.js

```diff
@@ -64,6 +64,18 @@ const LinkPage = () => (
     readme={linkReadme}
     examples={components}
     packageName="bpk-component-link"
+    usageTable={{
+      dos: [
+        'Use an "aria-label" for each link on a page should clearly describe where it will take you if you choose to follow it. E.g. "A link to the Skyscanner privacy policy."',
```

Comment:
Need to change this to not mention `aria-label` as the link text itself should be descriptive

Reply:
I guess there's also the point that if the link only contains an icon, that icon needs to have a suitable `alt` text to ensure it works for AT (and users on low bandwidth which might prevent the icon showing)